src/components/TodoApp.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { useLocalStorage } from '../utils/useLocalStorage';
+import { Footer } from './Footer';
+import { Header } from './Header';
+import { TodoList } from './TodoList';
+// import { ErrorNotification } from './ErrorNotification';
+import { Error } from '../types/Error';
+
+export const TodoApp: React.FC = () => {
+  const [todos, setTodos] = useLocalStorage('todos', []);
+  // const [errorMessage, setErrorMessage] = useState<string>('');
+  const [hidden, setHidden] = useState(true);
+  const [error, setError] = useState('');
+  const closeErrorMessage = () => {
+    setHidden(false);
+  };
+
+  const addTodo = (todo: Todo) => {
+    if ((todo.title).trim() && todo) {
+      setTodos([...todos, todo]);
+    } else {
+      setHidden(false);
+      setError(Error.EMPTY);
+    }
+
+    return todo;
+  };
+
+  useEffect(() => {
+    setTimeout(() => {
+      setError('');
+      closeErrorMessage();
+    }, 3000);
```

Comment:
You should clear a timeout on the component unmount 
https://bobbyhadz.com/blog/react-cleartimeout